docs/faq.md

```diff
@@ -97,6 +97,55 @@ the code is written to get a better grasp of it?
 There you will find documentation on everything from how our code is structure,
 how we aim to write our documentation, CI/CD, and more._
 
+---
+
+**Q:** What can I do, when my `api.search(...)` fails with a `cript.nodes.exception.CRIPTJsonDeserializationError` or similar?
```

Comment:
FAQ to explain to users of how to recover from bad search results.
(Hopefully rarely needed.)
But good to have for debugging anyways.